fuzzers/074-dump_all/Makefile

```diff
@@ -2,9 +2,9 @@
 N := 1
 SPECIMENS := $(addprefix build/specimen_,$(shell seq -f '%03.0f' $(N)))
 SPECIMENS_OK := $(addsuffix /OK,$(SPECIMENS))
-MAX_VIVADO_PROCESS ?= 4
-MAX_TILES_INSTANCE ?= 300
-MAX_NODES_INSTANCE ?= 30000
+MAX_VIVADO_PROCESS ?= 8
+MAX_TILES_INSTANCE ?= 600
+MAX_NODES_INSTANCE ?= 60000
```

Comment:
Are you sure about increasing this? Vivado has some kind of memory leak and/or slowdown as this number gets higher.

Reply:
Fixed